web/html/src/manager/maintenance/maintenance-windows.css

```diff
@@ -0,0 +1,14 @@
+:global(.list-tag-link) {
```

Comment:
Generic comment about style: please use classes defined in `.less` files in the `/branding/css/*` package as much as you can, especially when it comes to colors. As you know the branding can change from time to time and it would be very hard to go through all the customized CSS files in the codebase. Use dedicated CSS files only when it is really necessary for some specific and not reused behavior, otherwise better to create classes and reuse them, so the style of the whole application is consistent. Thanks